analytic_brand/views/res_brand.xml

```diff
@@ -6,7 +6,7 @@
         <field name="model">res.brand</field>
         <field name="inherit_id" ref="brand.res_brand_form_view" />
         <field name="arch" type="xml">
-            <xpath expr="//sheet" position="inside">
+            <xpath expr="//form" position="inside">
```

Comment:
why this change?

Reply:
sheet tag does not use in a parent that's why I use a form instead of sheet.